BugBountyFE/src/app/app.component.spec.ts

```diff
@@ -26,10 +26,10 @@ describe('AppComponent', () => {
     expect(app.title).toEqual('BugBountyFE');
   });
 
-  it('should render title', () => {
-    const fixture = TestBed.createComponent(AppComponent);
-    fixture.detectChanges();
-    const compiled = fixture.nativeElement;
-    expect(compiled.querySelector('.content span').textContent).toContain('BugBountyFE app is running!');
-  });
+  // it('should render title', () => {
```

Comment:
You do intend to uncomment everything later, yes?

Reply:
It was the default tests so I went ahead and removed it.